ci/create_initial_account.py

```diff
@@ -12,19 +44,37 @@ async def insert(tx):
                 return None
             return row['id']
 
+        hail_credentials_secret_name = f'{username}-gsa-key'
```

Comment:
Should this be None if we're in default? This is a change in behavior from the current code.

Reply:
You're right, this is different. I'll keep the same behavior as default.